hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/dstable/DistributionAddUpdateWindowLayout.java

```diff
@@ -94,8 +94,8 @@
     @Autowired
     private transient TenantMetaDataRepository tenantMetaDataRepository;
 
-    private Button saveDistribution;
-    private Button discardDistribution;
+    private Button saveDistributionButton;
```

Comment:
rename the button to `saveDistributionBtn` like it has been changed in the `AddUpdateRolloutWindowLayout.java
`for the `saveRolloutBtn`

Either use the word `Button` or `Btn` for consistency